content/tracing/setup/_index.md

```diff
@@ -52,23 +52,23 @@ apm_config:
+| ------------------------- | ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `enabled`                 | boolean     | When set to `true`, the Datadog Agent accepts trace metrics. Default value is `true`.                                                                              |
+| `apm_dd_url`              | string      | Datadog API endpoint where traces are sent.                                                                                                                        |
+| `env`                     | string      | Default environment to which traces should be registered under (e.g. *staging*, *production*, etc..).                                                              |
+| `extra_sample_rate`       | float       | Use this setting to adjust the trace sample rate. The value should be a float between `0` (no sampling) and `1` (normal sampling). Default value is `1`.           |
+| `max_traces_per_second`   | float       | Maximum number of traces to sample per second. Set to `0` to disable the limit (*not recommended*). The default value is `10`.                                     |
+| `ignore_resources`        | list        | A list of resources that the Agent should ignore.                                                                                                                  |
+| `log_file`                | string      | Location of the log file.                                                                                                                                          |
+| `replace_tags`            | list        | A list of tag replacement rules. See the [Scrubbing sensitive information](#scrubbing-sensitive-information) section.                                              |
+| `receiver_port`           | number      | Port that the Datadog Agent's trace receiver listen on. Default value is `8126`.                                                                                   |
+| `apm_non_local_traffic`   | boolean     | Allows the Agent to receive outside connections. It then listen on all interfaces.                                                                                 |
+| `max_memory`              | float       | Maximum memory that the Agent is allowed to occupy. When this is exceeded the process is killed.                                                                   |
+| `max_cpu_percent`         | float       | Maximum CPU percentage that the Agent should use. The Agent automatically adjusts its pre-sampling rate to stay below this number.                                 |
+| `max_connections`         | number      | Maximum number of network connections that the Agent is allowed to use. When this is exceeded the process is killed.                                               |
```

Comment:
This is an option we deprecated, we'd like to remove it from the public documentation.

Reply:
Same with `DD_CONNECTION_LIMIT`

Reply:
@l0k0ms can we remove them?

Reply:
Created https://github.com/DataDog/datadog-trace-agent/pull/469 to ensure this is clear moving forward

Reply:
https://github.com/DataDog/documentation/pull/3123